### PR TITLE
Update FS-1028-Implement Async.StartImmediateAsTask.md

### PR DESCRIPTION
**Pull request by OnurGumus**
Many formatting and grammatical fixes.

### CODE REVIEW

File: RFCs/FS-1028-Implement Async.StartImmediateAsTask.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
+The design suggestion, **Implement Async.StartImmediateAsTask** (https://github.com/fsharp/fslang-suggestions/issues/521) has been marked "approved in principle".
 This RFC covers the detailed proposal for this suggestion.
 
 * [x] Approved in principle
 * [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/521)
 * [ ] Details: [under discussion]
 * [ ] Implementation: [In progress]
 
-
+w
```
Comment (neoeinstein) [inline RFCs/FS-1028-Implement Async.StartImmediateAsTask.md]: Looks like an extra character snuck in.

File: RFCs/FS-1028-Implement Async.StartImmediateAsTask.md [hunk 2]
```diff
@@ -42,22 +41,24 @@ Example code:
...
-2- 4
-3- 4
-4- 4
-
-Where as using StartAsTask would yield this:
-1- 3
-2- 4
-3- 4
-4- 3
+If a synchronization context exists, the above code should return the same thread number. For applications like console, then thread 2. 3 and 4 will return the same id.
```
Comment (neoeinstein) [inline RFCs/FS-1028-Implement Async.StartImmediateAsTask.md]: `.` -> `,`

**Comment by neoeinstein** [comment]
A few small things. Also, when referring to functions or elements of code, use backticks (`) instead of making the names bold.

**Comment by OnurGumus** [comment]
Ok I added the fixes to the patch-1 branch. I am a little bit confused about where that branch came from though :)

**Comment by PatrickMcDonald** [comment]
@OnurGumus When you use the browser to edit a file in GitHub, GitHub forks the repo and creates a branch for you

**Comment by neoeinstein** [comment]
This looks good. You only need to use one backtick for inline monospacing, but Github looks to render it fine as is.

**Pull request merged**